### PR TITLE
I18n sync Down & Out 08/20

### PR DESCRIPTION
**Pull request by deploy-code-org**

### CODE REVIEW

**Comment by bethanyaconnor** [comment]
Why is 978e68b0a04d98b91f80f68ad63332e0b409d939 in the down/out sync ooc?

File: pegasus/cache/i18n/it-IT.yml [hunk 1]
```diff
@@ -2332,8 +2340,33 @@ it-IT:
...
+    are available for students from kindergarten through 8th grade in multiple locations
+    in the US for summer 2019.
+  hoc2018_dance_amazon_try_coding_this_summer_button: Join a camp
+  hoc2018_dance_amazon_scholarship_programs_title: Scholarship programs
+  hoc2018_dance_amazon_scholarship_programs_description: Amazon is offering scholarships
+    of $10,000 per year for underserved students who have successfully completed or
+    are currently enrolled in an AP CS course and intend to study a CS degree at a
+    4-year college or university in the United States.
   hoc2018_dance_amazon_scholarship_programs_button: Applica ora
+  hoc2018_dance_amazon_internships_title: Come Intern at Amazon
```
Comment (bethanyaconnor) [inline pegasus/cache/i18n/it-IT.yml:2362]: These are all approved translations that weren't actually translated. It kind of seems like this is on purpose, potentially as these strings are US-centric?

**Review by bethanyaconnor — APPROVED** [review]

File: pegasus/cache/i18n/it-IT.yml (hunk 1 above)
Comment (Hamms) [reply pegasus/cache/i18n/it-IT.yml:2362]: That does seem super weird. It was Francesco who did it, though, so I'm going to assume he had a good reason.

**Review by Hamms — APPROVED** [review]

File: pegasus/cache/i18n/it-IT.yml (hunk 1 above)
Comment (nkiruka) [reply pegasus/cache/i18n/it-IT.yml:2362]: If I remember correctly, the amazon internship information is only shown when the selected locale is US.

**Pull request merged**